src/client/components/Sorting/Sorting.component.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import './Sorting.styles.css';
+import { options } from './helper';
+import PropTypes from 'prop-types';
+
+const sortProducts = (sortMode, products) => {
+  if (sortMode === 'AlphabeticallyAZ') {
+    return products.sort((a, b) => a.name.localeCompare(b.name));
+  }
+  if (sortMode === 'AlphabeticallyZA') {
+    return products.sort((a, b) => b.name.localeCompare(a.name));
+  }
+
+  if (sortMode === 'created-at') {
+    return products.sort(
+      (a, b) => new Date(b.created_date) - new Date(a.created_date),
+    );
+  }
+};
+
+export default function Sorting({ products, categories }) {
+  const [sortMode, setSortMode] = React.useState(undefined);
```

Comment:
The initial sorting mode will be Alphabet A-Z. You can reuse const i taked about in previuos comment SORT_NAME_AZ